scriptmodules/supplementary/splashscreen.sh

```diff
@@ -12,7 +12,7 @@
 rp_module_id="splashscreen"
 rp_module_desc="Configure Splashscreen"
 rp_module_section="main"
-rp_module_flags="noinstclean !x86 !osmc !xbian !mali"
+rp_module_flags="noinstclean !all rpi !osmc !xbian"
```

Comment:
`!all` should be first (2) ?

Reply:
It needs to be before the other platform flags. Should be ok?

Ideally I'd like to move the "is module supported" flags separate from the others but it may break 3rd party packages. I may reorg when I do the backend selection code.

Reply:
Unless I've misunderstood?

Reply:
No, I though it should be _first_ - it's my misunderstanding.